.github/ISSUE_TEMPLATE/01-compatible-certification-request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Compatible Certification Request
+about: Start a request for a compatible certification 
+title: '[MP Component] [Version] Compatible Certification Request'
+label: 'Certification'
+assignees: ''
+---
+
+- [ ] Organization Name ("Organization") and, if applicable, URL:<br/>
+  // Add here
+- [ ] Product Name, Version and download URL (if applicable):<br/>
+  // Add here
+- [ ] Specification Name, Version and download URL:<br/>
+   // Add here
+- [ ] TCK Version, digital SHA-256 fingerprint and download URL:<br/>
```

Comment:
I think we decided that we don't need this for now, so probably remove it? Or state that it is optional?

Reply:
ok, I made it optional for now.